src/components/QueryFileUploader/utils/generateFileDefinitionBody.js

```diff
@@ -0,0 +1,6 @@
+import { convertBytesToKilobytes } from '@folio/stripes-data-transfer-components';
+
+export const generateFileDefinitionBody = file => ({
```

Comment:
It's used in one place and is not covered by unit tests, for example. Do we really need to move these several lines  to the separated module?

Reply:
Yeah, I prefer to have this kind of logic in separated module as it is implementation detail of the request to the server, so the component knows less about this kind of stuff. I think it is actually a good idea to cover it with tests then. Will do.